src/test/java/org/jenkinsci/plugins/workflow/cps/DSLTest.java

```diff
@@ -314,6 +315,91 @@ public void namedSoleParamForStep() throws Exception {
         p.setDefinition(new CpsFlowDefinition("try {def c = classLoad(getClass().name); error(/did not expect to be able to load ${c} from ${c.classLoader}/)} catch (ClassNotFoundException x) {echo(/good, got ${x}/)}", false));
         r.assertBuildStatusSuccess(p.scheduleBuild2(0));
     }
+
+    /**
+    * Tests the ability to execute a user defined closure
+    */
+    @Test public void userDefinedClosureInvocationExecution() throws Exception {
+        WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "p");
+        p.setDefinition(new CpsFlowDefinition("binding[\"my_closure\"] = { \n" +
+                                              " sleep 1 \n" + 
+                                              " echo \"my closure!\" \n" + 
+                                              "}\n" + 
+                                              "my_closure() ", false));
+        WorkflowRun b = r.assertBuildStatusSuccess(p.scheduleBuild2(0));
+        r.assertLogContains("my closure!", b); 
+    }
+    
+    /**
+    * Tests the ability to resolve DSL steps within a user defined closure
+    */
+    @Test public void userDefinedClosurePwdInvocationExecution() throws Exception {
+        WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "p");
+        p.setDefinition(new CpsFlowDefinition("my_closure = { \n" +
+                                              " sleep 1 \n" + 
+                                              " node{ \n" + 
+                                              "   echo pwd() \n" + 
```

Comment:
Uh, I'm not really sure the pwd() is needed/helpful there?  A simple echo 'some stuff' would probably suffice.

Reply:
ah, just saw this.  I can modify it.  I though echo was defined in CpsScript and was trying to test that DSL steps resolve within user defined closures - but that's not the case. 

I can remove this test as it isn't testing anything additional.